tests/shared.py

```diff
@@ -106,9 +116,10 @@ def create_train_fixtures(model="open_lm_11m", fsdp=False):
     # create base models
     random_seed()
     if fsdp:
+        init_distributed_device(args)
         with torch.device("meta"):
             model = create_model(args)
-        model = FSDP(model)
+            model = FSDP(model)
```

Comment:
Hm, was this necessary? I think actually the FSDP() call should be outside the meta device, so that FSDP materializes the tensors. 

Though really, since we've reverted the meta tensor commit, we should really just remove the `with torch.device("meta")` call here. I'll add an issue for this for a separate PR

Reply:
Actually, can you go ahead and remove the `with torch.device("meta")` call while you're at it? It shouldn't break anything. If it makes some tests fail, feel free to leave it in for now and we can revisit later.